judge/forms.py

```diff
@@ -121,3 +123,19 @@ def __init__(self, *args, **kwargs):
     def _has_social_auth(self, key):
         return (getattr(settings, 'SOCIAL_AUTH_%s_KEY' % key, None) and
                 getattr(settings, 'SOCIAL_AUTH_%s_SECRET' % key, None))
+
+
+class TOTPForm(Form):
+    TOLERANCE = getattr(settings, 'DMOJ_TOTP_TOLERANCE', 1)
```

Comment:
Setting should be self-documenting by specifying units (I'm assuming it's minutes).

Reply:
It's the number of TOTP time units, i.e. 30 seconds by default.